lib/swagger/index.js

```diff
@@ -17,7 +17,8 @@ swagger._getSwaggerBase = () => {
   let basePath, host, protocol
   if (jsonApi._apiConfig.urlPrefixAlias) {
     const urlObj = url.parse(jsonApi._apiConfig.urlPrefixAlias)
-    basePath = urlObj.pathname.replace(/\/$/, '')
```

Comment:
If we use the `(?<!^)\/$` regexp (with a look behind to assert we're not at the beginning of the string) above we can avoid the `if`.